draft-ietf-quic-recovery.md

```diff
@@ -1518,20 +1522,20 @@ newly acked_packets from sent_packets.
    OnPacketsAcked(acked_packets):
      for (packet in acked_packets):
        // Remove from bytes_in_flight.
-       bytes_in_flight -= packet.size
+       bytes_in_flight -= packet.sent_bytes
        if (InCongestionRecovery(packet.time_sent)):
          // Do not increase congestion window in recovery period.
-         return
+         continue
```

Comment:
I believe these 3 `return` are correct, not `continue`. L1523 says "Do not increase congestion window in recovery period."

Reply:
It is guaranteed that all acked packets satisfy `InCongestionRecovery`? If so, this first `return` is OK.

Reply:
To avoid confusion, `return` should not be used in `for`. `return` should be used before `for` by checking a specific packet.

Reply:
`bytes_in_flight -= packet.sent_bytes` should be evaluated for every packet, right? If so, `return` is simply wrong.